packages/nuxt-locokit/src/runtime/pages/auth/login.vue

```diff
@@ -0,0 +1,83 @@
+<template>
```

Comment:
Why this new file while SignIn already exists?

Reply:
Mistake from me... Removed